### ISSUE TITLE
Colliding name 'Object365' and 'object365'

### ISSUE THREAD

**Issue by hey24sheep** [fcakyon/yolov5-pip#72]
I am facing this issue in multiple places, is there any workaround for this?

```log
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'yolov5/data/Objects365.yaml'
  'yolov5/data/objects365.yaml'
```

**Comment by fcakyon** [comment]
@hey24sheep as a temporary solution, you can change the name or path of your yaml file.

**Comment by hey24sheep** [comment]
Actually, I am having this issue on git clone itself. I just want to add your library as an offline dataset on Kaggle. Even on windows if you download the ".zip" of your repo, it will give error on unzipping because of same name.

**Comment by hey24sheep** [comment]
Also, if you git clone your repository, it won't copy the "Objects365.yaml" (with capital O).

**Comment by fcakyon** [comment]
It is better to use it via `pip install yolov5` instead of cloning it (if you are not into developing it)

**Comment by hey24sheep** [comment]
Okay, I have fixed the issue on my end by deleting the 'Object365.yaml' file on a different [branch](https://github.com/hey24sheep/yolov5-pip/tree/fixed_repo) file on my fork.

I have used it as that but, I need it for an offline/local build so, I cannot do pip install on the machine. Thanks a lot for your help. But, I will still say, on Windows same name files give issues. Thanks a lot again for help and this package :)
